app/scripts/utils/visit-positioned-tracks.js

```diff
@@ -0,0 +1,19 @@
+import visitTracks from './visit-tracks';
+
+const visitPositionedTracks = (
+  positionedTracks,
+  visitor,
+  inclCombinedTracks = true
+) => {
```

Comment:
Would you mind adding a short docstring? `inclCombinedTracks` is a little difficult for me to understand. If it's false, all subchildren of the combined tracks are skipped? Why would it ever be false?